src/components/NftPreview/NftPreviewLabel.tsx

```diff
@@ -1,23 +1,30 @@
 import colors from 'components/core/colors';
 import styled from 'styled-components';
 import { BodyRegular } from 'components/core/Text/Text';
-import { Nft } from 'types/Nft';
 import breakpoints from 'components/core/breakpoints';
 
 type Props = {
-  nft: Nft;
+  nft: {
+    name?: string | null;
+    token_collection_name?: string | null;
+  };
```

Comment:
updated this to be more resilient to errors. backwards compatible with other use cases.